src/lib/github/newIssue.ts

```diff
@@ -0,0 +1,28 @@
+import * as R from "ramda";
+import formattedIssueComment from "./issue-presenter";
+import { Context } from "./types";
+import { RepoIssue } from "../parser";
+
+interface Issue {
+  title: string;
+  body: string;
+}
+
+export default async function getNewIssuesFromSource(context: Context, data: RepoIssue[]): Promise<Issue[]> {
```

Comment:
Could you make this function into 2. Such that we have one that make API calls (impure) and makes it available to a second function that manipulates the data. The function that does only data manipulation should have tests.